test/lib/heap.c

```diff
@@ -119,7 +131,7 @@ void test_heap_tear_down(struct raft_heap *h)
 {
     struct heap *heap = h->data;
     if (heap->n != 0) {
-      //munit_errorf("memory leak: %d outstanding allocations", heap->n);
+        // munit_errorf("memory leak: %d outstanding allocations", heap->n);
```

Comment:
Is this left here on purpose?

Reply:
Yeah it's left on purpose. That assertion is currently broken, it's basically a test-level issue.

Reply:
Leaks in tests are mainly detected using the memory sanitizer.